### ISSUE TITLE
EventBridge functions are not working

### ISSUE THREAD

**Issue by Devids** [paws-r/paws#426]
It looks like eventbridge functions are not working. Every EventBridge function I tried returns the following:
`Error: InvalidSignatureException (HTTP 400). Credential should be scoped to correct : 'events'.`

Simple example for reproduction:
```
eventbridge <- paws::eventbridge()
eventbridge$list_event_buses()
```

I think events should be used as `service_name` instead of eventbridge in `eventbridge_service.R`:

```
.eventbridge$metadata <- list(
   service_name = "eventbridge",
   endpoints...
```

**Comment by adambanker** [comment]
Thanks for submitting this issue! We updated the endpoint earlier this year to use `events` instead of `eventbridge`, but based on what you saw, it looks like we may need to change it somewhere else. I don't think we want to change the `service_name` field, though, since this is for the eventbridge service. I will look more into the issue.

**Comment by davidkretch** [comment]
I'm uploading an updated version of Paws to CRAN where this is working. It should be available this week. Sorry for the delay!

**Comment by davidkretch** [comment]
The fix is now available in the version of paws on CRAN (0.1.12). Sorry again for the delay.
